packages/tabs/src/views/Tab.js

```diff
@@ -28,6 +28,16 @@ const Tab = styled.div.attrs(props => ({
     [TabStyles['is-selected']]: props.selected
   })
 }))`
+  &&& {
+    ${props =>
+      !props.focused &&
+      `
+      :before {
+        box-shadow: none;
```

Comment:
Is this fixing a bug? Why is this included?

Reply:
Feels like this hack should be commented in code.

Reply:
I've added a comment for this.